subjects/countstars/README.md

```diff
@@ -0,0 +1,33 @@
+## CountStars
+
+### Instructions
+Write a function named `CountStar` that makes you fall asleep by counting stars, it takes an integer in the parameter
+- If the number is negative or equal to 0, return "`No star`"
```

Comment:
yours solution returns No Stars in plural

Reply:
Done